public/app/plugins/panel/timeseries/TimeSeriesPanel.tsx

```diff
@@ -47,10 +52,14 @@ export const TimeSeriesPanel: React.FC<TimeSeriesPanelProps> = ({
       onLegendClick={onLegendClick}
       onSeriesColorChange={onSeriesColorChange}
     >
-      <TooltipPlugin mode={options.tooltipOptions.mode as any} timeZone={timeZone} />
+      <TooltipPlugin mode={options.tooltipOptions.mode} timeZone={timeZone} />
       <ZoomPlugin onZoom={onChangeTimeRange} />
       <ContextMenuPlugin timeZone={timeZone} replaceVariables={replaceVariables} />
-      {data.annotations ? <ExemplarsPlugin exemplars={data.annotations} timeZone={timeZone} /> : <></>}
+      {data.annotations ? (
+        <ExemplarsPlugin exemplars={data.annotations} timeZone={timeZone} getFieldLinks={getFieldLinks} />
+      ) : (
+        <></>
+      )}
```

Comment:
uhm, is this correct (data.annotations)?

Reply:
It is. Because of this https://github.com/grafana/grafana/blob/32758e7cf0f9118f8f301a7720c8ad1b6cb237e0/public/app/features/query/state/runRequest.ts#L62-L63